src/controllers/auth.js

```diff
@@ -0,0 +1,24 @@
+const User = require('../models/User');
```

Comment:
[Naming / File Structure]
Since the controller is the user controller, I'd name it as such with `user.js` rather than `auth.js`. Technically we aren't even doing auth inside of here, we'll do that in a middleware so that is probably what we should name `auth.js`

Reply:
I renamed auth into user as recommended.